CMakeLists.txt

```diff
@@ -1,14 +1,9 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.7)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED on)
-include(CMakeDependentOption)
-
-project(EasyRPG_Player CXX C)
-set(PACKAGE_VERSION "0.5.3")
+project(EasyRPG_Player VERSION 0.5.3 LANGUAGES CXX C)
 
 # Source Files
-set(PLAYER_SRCS
+add_library(${PROJECT_NAME}_Static
```

Comment:
Does cmake still not support having different kinds of things (executable, library) with the same name? :-1: Maybe change the name to `player_aux` or something.
Also, this might be a shared library without explicit setting `STATIC`, no?

Reply:
is a static library by default because BUILD_SHARED_LIBS is OFF by default (strange default). But you are right, must force STATIC here.

Is not possible to have two targets with the same name but you can cheat:
https://gist.github.com/jlgerber/eafc4ee2b9954e27dd2bb009496b1b03#file-cmakelists-txt-L13
Will do this to get rid of _Static.

Reply:
nice, you are already using half of the cheat currently :+1: